### PR TITLE
Playground Auth without Supabase example

### PR DESCRIPTION
**Pull request by kien-ngo**
## PR-Codex overview
The focus of this PR is to enhance authentication functionality using Thirdweb Client. 

### Detailed summary
- Updated `client.ts` to use `THIRDWEB_SECRET_KEY` for client creation
- Modified authentication logic in `auth-button.tsx`
- Added `BasicAuthPreview` and `GatedContentPreview` components
- Updated environment variables in `.env.example`
- Improved authentication actions in `actions/auth.ts`
- Enhanced authentication page layout in `page.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/playground-web/src/app/connect/auth/actions/auth.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { THIRDWEB_CLIENT } from "@/lib/client";
+import { cookies } from "next/headers";
+import { type VerifyLoginPayloadParams, createAuth } from "thirdweb/auth";
+import { privateKeyToAccount } from "thirdweb/wallets";
+
+// FIXME TODO Somehow Vercel is not able to get the `THIRDWEB_ADMIN_PRIVATE_KEY` during build
+// should never have to do this (im sorry)
+const privateKey =
+  process.env.THIRDWEB_ADMIN_PRIVATE_KEY ||
+  process.env.NEXT_PUBLIC_THIRDWEB_ADMIN_PRIVATE_KEY;
```
Comment (joaquim-verges) [inline apps/playground-web/src/app/connect/auth/actions/auth.ts:11]: How about we just generate an account in the fallback case

Comment (kien-ngo) [reply apps/playground-web/src/app/connect/auth/actions/auth.ts:11]: yoooo that is a great idea

File: apps/playground-web/src/app/connect/auth/page.tsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+      <CodeExample
+        preview={<BasicAuthPreview />}
+        code={`"use client";
+
+import {
+  generatePayload,
+  isLoggedIn,
+  login,
+  logout,
+} from "@/app/connect/auth/actions/auth";
```
Comment (joaquim-verges) [inline apps/playground-web/src/app/connect/auth/page.tsx:99]: Make sure it's clear those are server actions

Comment (kien-ngo) [reply apps/playground-web/src/app/connect/auth/page.tsx:99]: It's done

File: apps/playground-web/src/app/connect/auth/page.tsx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+        <p className="max-w-[600px]">
+          Protect your page with thirdweb Auth. Deliver exclusive content to
+          users who qualify.
+        </p>
+      </div>
+
+      <CodeExample
+        preview={<GatedContentPreview />}
+        code={`import { THIRDWEB_CLIENT } from "@/lib/client";
+import { cookies } from "next/headers";
```
Comment (joaquim-verges) [inline apps/playground-web/src/app/connect/auth/page.tsx:143]: You changed the code here? I actually liked it better when you showed the jwt parsing - needs be clear what's going on from reading the snippet

File: apps/playground-web/src/app/connect/auth/page.tsx [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+  return (
+    <ConnectButton
+      client={THIRDWEB_CLIENT}
+      auth={{
+        isLoggedIn: async () => {
+          const authResult = await isLoggedIn();
+          if (!authResult) return false;
+          return true;
+        },
+        doLogin: async (params) => await login(params),
```
Comment (joaquim-verges) [inline apps/playground-web/src/app/connect/auth/page.tsx:113]: Add a comment that this is a server call

**Review by joaquim-verges — APPROVED** [review]

**Pull request merged**